### PR TITLE
Add entity_new field to Process model

### PR DESCRIPTION
**Pull request by JureZmrzlikar**

### CODE REVIEW

File: resolwe/flow/management/commands/register.py [hunk 1]
```diff
@@ -161,6 +161,7 @@ def register_processes(self, process_schemas, user, force=False, verbosity=1):
                 p['entity_type'] = p['entity']['type']
                 p['entity_descriptor_schema'] = p['entity'].get('descriptor_schema', p['entity_type'])
                 p['entity_input'] = p['entity'].get('input', None)
+                p['entity_new'] = p['entity'].get('new', False)
```
Comment (mstajdohar) [inline resolwe/flow/management/commands/register.py]: I'd rather have a more verbose name, e.g. `entity_always_create`

**Comment by mstajdohar** [comment]
Add a change log entry.

File: resolwe/flow/management/commands/register.py (hunk 1 above)
Comment (JureZmrzlikar) [reply resolwe/flow/management/commands/register.py]: Good point, done.

**Comment by JureZmrzlikar** [comment]
@dblenkus Please review and merge if OK, i would like to use this in the downstream package in the following release cycle.

**Pull request merged**